salt/pillar_roots/site.j2

```diff
@@ -28,33 +28,35 @@
       'ipdomain': 'example.com',
       'timezone': 'America/Denver',
 
-      'enabled_backends': 'lvm, cinder, ceph',
+      'enabled_backends': 'lvm, cinder, ceph, nfs, netapp_ontap_san',
       'lvm_poolname': 'opensds-volumes',
       'ceph_poolname': 'ceph-volumes',
       'cinder_poolname': 'cinder-volumes',
       'dorado_poolname': 'dorado-volumes',
       'fusionstorage_poolname': 'fusionstorage-volumes',
 
-      'gelato_release': 'v0.6.0',
+      'apiserver_release': 'v0.10.0',
+
+      'gelato_release': 'v0.10.0',
       'gelato_service': 'multi-cloud',
-      'gelato_uri': 'https://github.com/opensds/multi-cloud/releases/download',
-      'gelato_hashsum': '92a26c93a8b56472922d7b771eb41f3689e585e87aaee6466dae5684f14976ea',
+      'gelato_hashsum': 'c2edc201d3bb89f60e09bf1cc8ca5f94861f6163b0d59abfff1ad05a43536550',
+      'gelato_uri': 'https://github.com/sodafoundation/multi-cloud/releases/download',
 
-      'hotpot_release': 'v0.6.0',
+      'hotpot_release': 'v0.10.1',
       'hotpot_service': 'opensds',
-      'hotpot_hashsum': '34916233b3f145491efd97108558938dadaa3f699f4508cf946e22f28aac4741',
-      'hotpot_uri': 'https://github.com/opensds/opensds/releases/download',
+      'hotpot_hashsum': 'ee53570846690ef660121079299992096545178816625eceaed3a0c9559f72e8',
+      'hotpot_uri': 'https://github.com/sodafoundation/opensds/releases/download',
```

Comment:
You may needed to rename hotpot_uri to 'https://github.com/sodafoundation/api/releases/download' to support latest releases